src/components/MovesTab/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { ScrollView, Text, View } from "react-native";
+
+import type { Pokemon } from "../../types/interfaces/Pokemon";
+
+interface MovesTabProps {
+    viewedPokemon: Pokemon;
+}
+
+export default function MovesTab({ viewedPokemon }: MovesTabProps) {
+    const removeHyphenAndMakeToTitleCase = (str) => {
+        const words = str.split("-");
+
+        const capitalizedWords = words.map(
+            (word: string) => word.charAt(0).toUpperCase() + word.slice(1)
+        );
```

Comment:
Think we already have a `capitalizeString` utility function, so you can call that instead of repeating the logic here.